priv/repo/migrations/20240612081138_add_auditing_settings.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Trento.Repo.Migrations.AddAuditingSettings do
```

Comment:
Optional: maybe a [constraint](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#constraint/3 ) on making retention period > 0:

Reply:
let's take this into account if we move back from a one-json-field to separate ones